### PR TITLE
show only open jobs / add G3

### PR DESCRIPTION
**Pull request by mojoaxel**

### CODE REVIEW

File: src/jobs.njk [hunk 1]
```diff
@@ -29,9 +29,13 @@ description: 'We provide fully remote developer jobs with meaningful projects.'
...
 <section class="open-positions">
+{% if (collections.jobs | openPositionFilter())|length > 0 %}
 	<h2 class="open-positions-title">Open Positions</h2>
 
-	{% for job in collections.jobs %}
+	{% for job in collections.jobs | openPositionFilter() %}
 		{% include 'layouts/job.njk' %}
 	{% endfor %}
+{% else %}
+	<h2 class="open-positions-title">There are no open positions at this moment!</h2>
```
Comment (josurzyn) [inline src/jobs.njk:39]: 'at the moment' sounds a bit more natural, but you can leave it if you want

File: src/jobs/angular-developer.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 ---
 id: angular-developer
-title: Angular Developer (EU, Remote)
+title: Angular Developer (Remote, EU only)
+open: false
```
Comment (josurzyn) [inline src/jobs/angular-developer.md:4]: I guess I've got some catching up to do!

File: src/jobs/fullstack-developer.md [hunk 3]
```diff
@@ -18,10 +19,10 @@ logos:
 - Experience using npm and git
 - Fluent in English
 - Sufficient self-organisation and work-time management
+- Angular and node.js experiance
```
Comment (josurzyn) [inline src/jobs/fullstack-developer.md:22]: experience

**Review by josurzyn — CHANGES_REQUESTED** [review]
The 'Join us' button and the adverts in the team section are still there, which seems a bit inconsistent with removing the link to the 'Jobs' page.

Excited to hear if we've hired, or just aren't looking any more 😅

File: src/jobs/angular-developer.md (hunk 2 above)
Comment (mojoaxel) [reply src/jobs/angular-developer.md:4]: I think we'll have one full and a half position filled at the end of next week :-)

File: src/jobs/angular-developer.md [hunk 4]
```diff
@@ -29,8 +38,8 @@ logos:
 
 ## Working Tools and conditions
 
-- fully remote
-- Possibility of using your choice of coworking space
+- 100% fully remote
+- Possibility of using your choice of coworking-space
```
Comment (josurzyn) [inline src/jobs/angular-developer.md:42]: coworking space doesn't need a hyphen

File: src/jobs/fullstack-developer.md [hunk 5]
```diff
@@ -18,19 +25,19 @@ logos:
...
 - C, C++ or Rust
 - Docker
 - Project management
 
 ## Working Tools and conditions
 
-- Fully remote
-- Possibility of using your choice of coworking space
+- 100% fully remote
+- Possibility of using your choice of coworking-space
```
Comment (josurzyn) [inline src/jobs/fullstack-developer.md:40]: no hyphen necessary

**Review by josurzyn — APPROVED** [review]
Not sure why all the checks failed, but otherwise LGTM!

**Comment by mojoaxel** [comment]
> Not sure why all the checks failed, but otherwise LGTM!

This was a netlify issue. The deprecated a build image we used.

**Pull request merged**